### ISSUE TITLE
Expert initials required

### ISSUE THREAD

**Issue by guyellis** [airpair/airpair-com-v1-issue-tracker#362]
When trying to reply to a job request:

```
Expert initials required

Err... sorry about this. Please tell us what happened.
```

**Comment by guyellis** [comment]
After I added initials to my profile the next error was *Location Require* - Adding initials to my profile fixed this.

**Comment by jkresner** [comment]
Yes - this flow will be fixed again as it's moved onto consult.airpair.com,
best Jonathon
ᐧ

On Mon, Oct 19, 2015 at 10:01 AM, Guy Ellis <notifications@github.com>
wrote:

> After I added initials to my profile the next error was *Location Require*
> - Adding initials to my profile fixed this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/airpair/airpair-com-v1-issue-tracker/issues/362#issuecomment-149282212>
> .
>



-- 
*Jonathon Kresner*
*Co-founder, CEO*
@AirPair <http://twitter.com/airpair> | *airpair.com
<http://www.airpair.com>*
@hackerpreneur <http://twitter.com/hackerpreneur>
